Authentication section for the new admin UI (#1514)
keycloak/keycloak-documentation: server_admin/topics/authentication/x509.adoc

[[_browser_flow]]
==== Adding X.509 client certificate authentication to browser flows

ifeval::[{project_community}==true]
. Click *Authentication* in the menu.

> . Select *Duplicate* from the "Action list" to make a copy of the built-in "Browser" flow.

Reviewer: ```suggestion
. Click the *Browser* flow.
. From the *Action* list, select *Duplicate*. 
```

==== Adding X.509 Client Certificate Authentication to a Direct Grant Flow

. Click *Authentication* in the menu.
ifeval::[{project_community}==true]
. Select *Duplicate* from the "Action list" to make a copy of the built-in "Direct grant" flow.
. Enter a name for the copy.
. Click *Duplicate*.
. Click the created flow.
. Click on the *🗑️* (trash can icon) of the "Username Validation" and click *Delete*.

> . Click on the *🗑️* (trash can icon) of the "Password" and click *Delete*.

Reviewer: Click the trash can icon 🗑️ of the,,,,
